bentoml/_internal/models/store.py

```diff
@@ -187,6 +186,24 @@ def export_model(self, name: str):
     def import_model(self, name: str):
         ...
 
+    def __getattribute__(self, item):
+        inherited = object.__getattribute__(self, item)
+        if "model" in item:
+
+            def wrapped_check_identifier(*args, **kwargs):
```

Comment:
let's just add a `validate_model_name` func call in get_model and register_model

Reply:
I was thinking to use it in every function since it will invoked the `name` args